src/content/reference/react/createElement.md

```diff
@@ -34,44 +34,44 @@ function Greeting({ name }) {
 }
 ```
 
-[See more examples below.](#usage)
+[Lihat lebih banyak contoh lainnya di bawah ini.](#usage)
 
 #### Parameters {/*parameters*/}
```

Comment:
```suggestion
#### Parameter {/*parameters*/}
```

Use the correct translation for API reference headings.

Reply:
updated